quickstart/201-function-app-key-vault-ref/main.tf

```diff
@@ -0,0 +1,114 @@
+data "azurerm_subscription" "current" {}
```

Comment:
I've seen this `201-function-app-key-vault-ref` in #114, is this by intention or by mistake?

Reply:
removed the folder, this is my mistake.